jsk_unitree_robot/cross/README.md

```diff
@@ -19,6 +19,11 @@ $ newgrp
 $ sudo apt install -y qemu-user-static
 ```
 
+For Mac Users, you need to install `python-vcstool`
+```
+$ brew install python-vcstool
+```
+
```

Comment:
I cannot execute `brew install python-vcstool` in my environment.
```
$ brew install python-vcstool
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
install-peerdeps                                                    redis@6.2                                                           zx

==> Tapping homebrew/cask
Cloning into '/opt/homebrew/Library/Taps/homebrew/homebrew-cask'...
remote: Enumerating objects: 647014, done.
remote: Counting objects: 100% (46/46), done.
remote: Compressing objects: 100% (34/34), done.
remote: Total 647014 (delta 26), reused 25 (delta 12), pack-reused 646968
Receiving objects: 100% (647014/647014), 306.06 MiB | 13.19 MiB/s, done.
Resolving deltas: 100% (458232/458232), done.
Tapped 4014 casks (4,086 files, 326.7MB).
Warning: No available formula with the name "python-vcstool". Did you mean python-yq?
==> Searching for similarly named formulae...
This similarly named formula was found:
python-yq
To install it, run:
  brew install python-yq
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
```
My environment
```
 base  ~  ros  jsk_robot  jsk_unitree_robot  cross  unitree  $  
$ brew --version
Homebrew 3.5.2
Homebrew/homebrew-core (git revision 50275198c04; last commit 2022-06-15)
Homebrew/homebrew-cask (git revision 4b9512207c; last commit 2022-06-15)

$ which brew
/opt/homebrew/bin/brew
```
Maybe `pip install vcstool`? or something else?